docs/Rakefile

```diff
@@ -15,12 +15,12 @@ ADOC_VARIABLES     = FileList.new('source/variables.adoc',
 )
 GENERATED_ADOC_DIR = "source/command-ref"
 ADOC_FILES = Rake::FileList.new do |list|
-	MARKDOWN_FILES.each do |f|
-        adoc = f.ext(".adoc")
-        adoc = File.split(adoc)[1]
-        adoc = File.join(GENERATED_ADOC_DIR, adoc)
-		list.add(adoc)
-	end
+  MARKDOWN_FILES.each do |f|
```

Comment:
Why these changes?

Reply:
@hferentschik There are inconsistencies in tabs alignment here.

Like https://github.com/minishift/minishift/blob/master/docs/Rakefile#L139-L143 

and 

https://github.com/minishift/minishift/blob/master/docs/Rakefile#L101-L116

and then this one https://github.com/minishift/minishift/blob/master/docs/Rakefile#L128-L136

These changes automatically got it when I was trying to fix the alignment here https://github.com/minishift/minishift/blob/master/docs/Rakefile#L19-L23.

And since, my Sublime is set to have tab with 2 spaces, other places it got aligned.

Bottom line is we now have tab alignment consistent in `docs/Rakefile` with two space tab.

Reply:
At least keep this in a different commit, please. I have no problem using the same issue key, but I'd like to distinguish the actual fix for the issue from this unrelated cleanup.

Reply:
Done